tests/dummy/app/templates/components-examples/flexberry-simpleolv/settings-example.hbs

```diff
@@ -47,7 +47,7 @@
         previousPage=(action "previousPage")
         gotoPage=(action "gotoPage")
         nextPage=(action "nextPage")
-        componentName="SOLVSettingsExampleObjectListView"
+        componentName="SOLVSettingExampleObjectListView"
```

Comment:
Why `SOLVSettingExampleObjectListView`? It was ...Setting**s**... like for `flexberry-objectlistvew`?